src/maven/orb.yml

```diff
@@ -125,7 +126,7 @@ commands:
     steps:
       - run:
           name: Generate Cache Checksum
-          command: find . -name 'pom.xml' -exec cat {} + | shasum | awk '{print $1}' > /tmp/maven_cache_seed
+          command: find . -name 'pom.xml' | sort | xargs cat > /tmp/maven_cache_seed
```

Comment:
Not sure why this shows up in the diff, but I see that this change is already both in `master` (https://github.com/CircleCI-Public/circleci-orbs/blame/4172b7a403dc6429e1b2479cf4c576f870524be2/src/maven/orb.yml#L129) and also in the published orb.